File: Problem_002.cpp
#include <iostream>
#include <string>

using namespace std;

int main (){

	long first = 1, second = 1, next, sum = 0;



	for(int i = 0; i <= 4000000; i++){
		next = first + second;
		first = second;
		second = next;

		if(next % 2 == 0){
			sum += next;
		}
		if(next > 4000000){
			break;
		}
	}

	cout << "The sum of fib elements under 4million that are even is: " << sum;


	return 0;
}

File: In_Progress/Problem_002.java
public class Problem002 {
	public static void main(String args[]){
		int first = 1, second = 1, third = 0;

		int sum = 0;

		while(third <= 4000000){
			third = first + second;
			first = second;
			second = third;

			sum += (third % 2 ==0) ? third:0;
		}

		System.out.println("The sum is: " + sum);
	}
}
